### PR TITLE
Update and edit CI Monitor page

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do?
  - Edit the CI Pipeline configuration instructions
  - Add common monitor types to the CI Test montior setup
  - Edit the CI Test configuration instructions
  - Change the size of images to 100%

### Motivation
[DOCS-4432](https://datadoghq.atlassian.net/browse/DOCS-4432)

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/monitors/create/types/ci.md [hunk 1]
```diff
@@ -32,64 +32,62 @@ To create a [CI monitor][2] in Datadog, use the main navigation: *Monitors -> Ne
...
-    * **Monitor over the `Job` level**: If the `Job` level is selected, the monitor will only include job events for evaluation, which represent the execution of a group of commands.
-    * **Monitor over the `Command` level**: If the `Command` level is selected, the monitor will only include manually instrumented [custom command][1] events for evaluation, which represent individual commands being executed in a job.
-    * **Monitor over all levels**: If the `All` level is selected, the monitor will include all types of events for evaluation.
-
+2. Select the CI Pipeline events level:
+    * **Pipeline**: Evaluates the execution of an entire pipeline, usually composed of one or more jobs.
+    * **Stage**: Evaluates the execution of a group of one or more jobs in CI providers that support it.
+    * **Job**: Evaluates the execution of a group of commands.
+    * **Command**: Evaluates manually instrumented [custom command][1] events, which are individual commands being executed in a job.
+    * **Alls**: Evaluates all types of events.
```
Comment (buraizu) [inline content/en/monitors/create/types/ci.md:48]:
```suggestion
    * **All**: Evaluates all types of events.
```

File: content/en/monitors/create/types/ci.md [hunk 2]
```diff
@@ -32,64 +32,62 @@ To create a [CI monitor][2] in Datadog, use the main navigation: *Monitors -> Ne
...
-4. Configure the alerting grouping strategy (optional):
-   * If the query has a `group by`, an alert is sent for every source according to the group parameters. An alerting event is generated for each group that meets the set conditions. For example, you could group a query by `@test.full_name` to receive a separate alert for each CI Test full name when the number of errors is high. Test full name is a combination of a test suite and test name, for example: `MySuite.myTest`. In Swift, test full name is a combination of a test bundle, and suite and name, for example: `MyBundle.MySuite.myTest`.
+1. Common monitor types: (optional) Provides a template query for each of the **New Flaky Test**, **Test Failures**, and **Test Performance** common monitor types, which you can then customize. Learn more about this feature by reading [Track new flaky tests](#track-new-flaky-tests).
+2. Construct a search query using the same logic as a CI Test explorer search. For example, you can search failed tests for the `main` branch of the `myapp` test service using the following query: `@test.status:fail @git.branch:main @test.service:myapp`
```
Comment (buraizu) [inline content/en/monitors/create/types/ci.md:77]:
```suggestion
2. Construct a search query using the same logic as a CI Test explorer search. For example, you can search failed tests for the `main` branch of the `myapp` test service using the following query: `@test.status:fail @git.branch:main @test.service:myapp`.
```

Comment (buraizu) [reply content/en/monitors/create/types/ci.md:77]: Not sure if this is left off intentionally or not, feel free to ignore this suggestion if you're hoping to leave it off to make the snippet easier to copy.

File: content/en/monitors/create/types/ci.md [hunk 3]
```diff
@@ -32,64 +32,62 @@ To create a [CI monitor][2] in Datadog, use the main navigation: *Monitors -> Ne
...
 
-{{< img src="monitors/monitor_types/ci_tests/define-the-search-query.png" alt="A query for CI Status:Error that is being set to group by Pipeline Name" style="width:80%;" >}}
+{{< img src="monitors/monitor_types/ci_tests/define-the-search-query.png" alt="A query for CI Status:Error that is being set to group by Pipeline Name" style="width:100%;" >}}
 
 #### Test runs with different parameters or configurations
-If you run tests with the same test full name, but different test parameters or configurations, it's recommended to use `@test.fingerprint` in the monitor group by. This way, alerts trigger for test runs with specific test parameters or configurations. Using `@test.fingerprint` provides the same granularity level as the Test Stats, Failed and Flaky Tests section on the **Commit Overview** page.
+Use `@test.fingerprint` in the monitor group by when you have tests with the same test full name, but different test parameters or configurations. This way, alerts trigger for test runs with specific test parameters or configurations. Using `@test.fingerprint` provides the same granularity level as the Test Stats, Failed, and Flaky Tests section on the **Commit Overview** page.
```
Comment (buraizu) [inline content/en/monitors/create/types/ci.md:90]:
```suggestion
Use `@test.fingerprint` in the monitor `group by` when you have tests with the same test full name, but different test parameters or configurations. This way, alerts trigger for test runs with specific test parameters or configurations. Using `@test.fingerprint` provides the same granularity level as the Test Stats, Failed, and Flaky Tests section on the **Commit Overview** page.
```

Comment (buraizu) [reply content/en/monitors/create/types/ci.md:90]: I know it's not part of your update, but just noticed this inconsistency

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks, just made a few minor suggestions

File: content/en/monitors/create/types/ci.md (hunk 3 above)
Comment (estherk15) [reply content/en/monitors/create/types/ci.md:90]: Thanks for catching this!

**Review by buraizu — APPROVED** [review]

**Pull request merged**